src/Command/RepublicanSilenceCloseEventCommand.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Entity\RepublicanSilence;
+use AppBundle\Event\EventCancelHandler;
+use AppBundle\Repository\CitizenActionRepository;
+use AppBundle\Repository\EventRepository;
+use AppBundle\RepublicanSilence\RepublicanSilenceManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class RepublicanSilenceCloseEventCommand extends Command
+{
+    protected static $defaultName = 'app:republican-silence:close-event';
+
+    private $manager;
+    private $eventRepository;
+    private $actionRepository;
+    private $eventCancelHandler;
+
+    public function __construct(
+        RepublicanSilenceManager $manager,
+        EventRepository $eventRepository,
+        CitizenActionRepository $actionRepository,
+        EventCancelHandler $eventCancelHandler
+    ) {
+        parent::__construct();
+
+        $this->manager = $manager;
+        $this->eventRepository = $eventRepository;
+        $this->actionRepository = $actionRepository;
+        $this->eventCancelHandler = $eventCancelHandler;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('This command closes each committee event or citizen action when it match republican silence criteria')
```

Comment:
`it match republican silence criteria` => `it matches a republican silence criteria`

Reply:
fixed 👍